src/cli/openstudio_cli.rb

```diff
@@ -55,6 +55,16 @@
 #OpenStudio::Logger.instance.standardOutLogger.setLogLevel(OpenStudio::Warn)
 OpenStudio::Logger.instance.standardOutLogger.setLogLevel(OpenStudio::Error)
 
+OpenStudio.autoload(:Airflow, ":/openstudio_init_extended.rb")
+OpenStudio.autoload(:EnergyPlus, ":/openstudio_init_extended.rb")
+OpenStudio.autoload(:GbXML, ":/openstudio_init_extended.rb")
+OpenStudio.autoload(:ISOModel, ":/openstudio_init_extended.rb")
+OpenStudio.autoload(:Measure, ":/openstudio_init_extended.rb")
```

Comment:
```suggestion
OpenStudio.autoload(:Measure, ":/openstudio_init_extended.rb")
OpenStudio.autoload(:Ruleset, ":/openstudio_init_extended.rb")
```

@lefticus @tijcolem 

This could be the reason the regression tests are failing. This is the old namespace where measures would live, and we do have one measure that uses that old one:

https://github.com/NREL/OpenStudio-resources/blob/a729a7c85f55cc05d7ba6d50c45d21b8621611c9/measures/gem_env_report/measure.rb#L5

```
class GemEnvironmentReport < OpenStudio::Ruleset::ModelUserScript
```

In newer measures this would be updated to 

``` 
class GemEnvironmentReport < OpenStudio::Measure::ModelMeasure
```

I am not sure how the `autoload` thing works, but if it's parsing stuff just by name it could be missing the needed init...

Reply:
This is typedef'ed into ruby here: https://github.com/NREL/OpenStudio/blob/ff0e898ff96e0653e1b72da4200a3403b927fdaa/src/utilities/core/RubyInterpreter.hpp#L687-L856

Reply:
Yeah, that's probably it: https://github.com/NREL/OpenStudio-workflow-gem/blob/3b5918b6a0d8891e245762600735abe729abe81b/lib/openstudio/workflow_runner.rb#L42

I haven't found the way to correctly autoload this. I've also tried inserting it here 

https://github.com/NREL/OpenStudio/blob/ff0e898ff96e0653e1b72da4200a3403b927fdaa/src/utilities/core/RubyInterpreter.hpp#L851

Reply:
Fixed in https://github.com/NREL/OpenStudio/pull/4028/commits/9d61e34d8f2cbd5693f969653f52344914ba4f27